### DISCUSSION TITLE
How to use scss as style language

### DISCUSSION THREAD

**Discussion by Julyzl** [FranckFreiburger/vue3-sfc-loader#189] [Q&A] [answered]
it's my example，But it didn't work，Can the author write an example，thanks！
import sassloader from 'sass-loader'
options: {
			moduleCache: {
				vue: Vue,
				dayjs: dayjs,
				scss: (source) => Object.assign(sass(source), { deps: () => [] }) // (source) => Object.assign(sass(source), { deps: () => [] })
			},
			// 获取文件
			async getFile(url: any) {
				const res = await fetch(url)
				if (!res.ok) throw Object.assign(new Error(url + ' ' + res.statusText), { res })
				return await res.text()
			},
			// 添加样式
			addStyle(textContent: any) {
				const style = Object.assign(document.createElement('style'), {
					textContent
				})
				const ref = document.head.getElementsByTagName('style')[0] || null
				document.head.insertBefore(style, ref)
			}
		}

**Comment by mustapha-wang** [comment]
what is wrong?错误信息是啥？

**Comment by Julyzl** [comment]
报错信息： vue3-sfc-loader.js?v=4c92af84:24805 Uncaught (in promise) TypeError: Unable to handle  files (scss)

**Comment by Julyzl** [comment]
![微信图片_20240321135557](https://github.com/FranckFreiburger/vue3-sfc-loader/assets/48594181/a5cee89a-e30c-474b-8f26-b3c6d0289509)
这是我的示例，这样写scss就不会报Unable to handle files (scss)这个错了

**Comment by Julyzl** [comment]
但是写lang=scss 没有效果，就像好像没写样式一样
你那边有示例吗？

**Comment by FranckFreiburger** [comment, marked as answer]
look at this example:
https://github.com/FranckFreiburger/vue3-sfc-loader/discussions/181
